api/client/commands.go

```diff
@@ -1656,13 +1657,35 @@ func (cli *DockerCli) CmdPs(args ...string) error {
 	}
 
 	w := tabwriter.NewWriter(cli.out, 20, 1, 3, ' ', 0)
+	if *quiet {
+		*fields = "c"
+	}
+
+	if *size {
+		*fields = *fields + "z"
```

Comment:
actually, do you purposely want to allow `--fields=cccc`  ?  Guess if that's what the user wants.....